client_test/directfund_integration_test.go

```diff
@@ -84,6 +84,10 @@ func TestDirectFundIntegration(t *testing.T) {
 			t.Fatal("Expected nonzero on chain funding, but got zero")
 		}
 
+		if _, channelStillInStore := store.GetChannelById(con.Id); channelStillInStore {
+			t.Fatalf("Expected channel to have been destroyed in %v's store, but it was not", store.GetAddress())
```

Comment:
```suggestion
			t.Fatalf("Expected the channel.Channel data to have been destroyed in %v's store, but it was not", store.GetAddress())
```

This suggestion tightly couples the error message to current package / struct names, but might be worth it to provide context.

The check could also be moved closer to the check that the ConsensusChannel (ledger channel) **does** exist, with a comment similar to the one introduced in this PR in `directdefund_integration_test.go`. Something like:

```
// Ensure that we have a consensus channel in the store
// And that we no longer have a regular channel
```